Makefile

```diff
@@ -49,7 +49,8 @@ STANDARDTESTFLAGS := -logtostderr
 TESTFLAGS         :=
 
 ifeq ($(STATIC),1)
-GOFLAGS  += -a -tags netgo -ldflags '-extldflags "-lm -lstdc++ -static"'
+GOFLAGS  += -tags netgo
```

Comment:
I verified that I ended up with a statically linked netgo package, but I probably have rebuilt the stdlib manually at some point in the past. I'll investigate but you're right, it seems there's something tricky in go1.4.
I'll investigate.